<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="screenFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QStackedWidget" name="displayWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="deviceOffPage"/>
     <widget class="QWidget" name="sessionPage">
      <widget class="QWidget" name="mainScreenWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>281</height>
        </rect>
       </property>
       <widget class="QProgressBar" name="treatmentBar">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>341</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLCDNumber" name="timeDisplay">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="plotWidgetContainer" native="true">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>89</y>
          <width>341</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="menuPage">
      <widget class="QFrame" name="newSessionFrame">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="newSessionLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;span style=&quot;font-size: x-large;&quot;&gt;NEW SESSION&lt;/span&gt;‍‍‍‍‍‍‍‍</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="sessionLogFrame">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="sessionLogLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;span style=&quot;font-size: x-large;&quot;&gt;SESSION LOG&lt;/span&gt;‍‍‍‍‍‍‍‍</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="timeAndDateFrame">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="dateTimeLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;span style=&quot;font-size: x-large;&quot;&gt;DATE &amp; TIME&lt;/span&gt;‍‍‍‍‍‍‍‍</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="sessionLogPage">
      <widget class="QListWidget" name="sessionListWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>321</width>
         <height>251</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="movement">
        <enum>QListView::Snap</enum>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLabel" name="SessionLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Session History</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="dateAndTimePage">
      <widget class="QPushButton" name="setTimeButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Set Date/Time</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="dateTimeEditLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Current Date Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/pause.png</normaloff>../Assets/pause.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/stop.png</normaloff>
      <normalon>../Assets/stop.png</normalon>../Assets/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="lostContactLight">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="treatmentLight">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="contactLight">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="menuNavDownButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="menuNavUpButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/power.png</normaloff>../Assets/power.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/menu.png</normaloff>../Assets/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="chargeBarWidget" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="chargeBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="chargeIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
